operator/cmd/operator/app/operator.go

```diff
@@ -110,6 +114,28 @@ func Run(ctx context.Context, o *options.Options) error {
 		return err
 	}
 
+	if err := manager.AddReadyzCheck("readyz", healthz.Ping); err != nil {
+		klog.Errorf("Unable to set up ready check: %v", err)
+		return err
+	}
```

Comment:
Can you explain why you need the readyz in addition to the healthy check?

Reply:
Thanks @RainbowMango 
`readyz` and `healthz` correspond to `readyzHandler` and `healthzHandler` in controller-runtime. 
Hmmm.... 🤔️ It may be necessary to refine it? 

ref: https://github.com/kubernetes-sigs/controller-runtime/blob/main/pkg/manager/internal.go#L97-L101

Reply:
> Should we introduce the two checks on all the other karmada components? I would suggest we can start another PR or issue to discuss that topic, and let this PR focus on metrics thing.

Got it.
Remove it frist.